tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -83,6 +89,14 @@ When('the user deletes the following elements using the webUI', function (table)
   return client.page.filesPage()
 })
 
+Given('the following files have been deleted', function (table) {
```

Comment:
Given steps should be implemented as API steps to save time. There are only for set-up, nothing we really want to test. If we want to test the deletion itself that would be a separate webUI step

Reply:
I see .. is there a similar step already in our tests here? THX

Reply:
`Given user {string} has favorited element {string}'`
https://github.com/owncloud/phoenix/blob/master/tests/acceptance/stepDefinitions/webdavContext.js#L37

Reply:
we already have the owncloud-sdk dependency in phoenix which is the js lib to talk to owncloud.

Any objections in using it instead of this webdav helper?
I smell code duplication ....